### ISSUE TITLE
Exhibit Categories

### ISSUE THREAD

**Issue by theo-armour** [exploratoria/exploratoria.github.io#8]
@exploratoria/owners 

Currently we are using the Exploratorium list of headings as our categorization system.

It might be nice to have more guidance as to where exhibits should be placed.

One example is that used by PIRA.I don't yet know where the actual spec is, but you can obtain a glimpse by clicking the menus here:

http://physicslearning.colorado.edu/ldl/

or a bit more detail here:

http://physicslearning.colorado.edu/bib/bibMain.asp

Are there other systems we should be looking at?

Or can we be really cool and organize things in whatever way you want:

You like Exploratorium? Bingo! Dewey Decimal? we gotcha covered! PIRO? Here ya go!

In any case, many exhibits will cover several areas, so a nice system - or just a plan of attack - for categories and tags would be cool.

So should we and/or how can we get started on this?

**Comment by paulmasson** [comment]
We need to have some initial categories to have an organized file system. What do you mean by using tags? How does one even attach a tag to a file on Github?

**Comment by theo-armour** [comment]
> > We need to have some initial categories to have an organized file system. 

Agreed. And, as we know, the current categories are based on the [Exploratorium list of exhibits](http://www.exploratorium.edu/explore).

The web site offers no background or explanation as to how they arrived at this list. Therefore one must assume it's based on exhibits that appear to have helped sell tickets. This is a certainly a good reason for them, but it may not be the reason for us.

So I think we could spend some time looking around for the shoulders of giants that we can stand on. Certainly [PIRA](http://physicslearning.colorado.edu/ldl/) is one avenue to explore. I wonder if there are others? For instance. how might a list of exhibits be structured in a German museum of science and technology?

> > What do you mean by using tags? How does one even attach a tag to a file on Github?

https://developer.github.com/v3/git/tags/

But those are perhaps too programmer oriented. We could be thinking more in [semantic web](https://en.wikipedia.org/wiki/Semantic_Web) terms - without betting bogged down in the semantics.

For example, you may note that in a number of the files that I have created recently, you will see a meta tag that might look something like this:

```
<meta name=keywords content='WebGL,Three.js,Tween.js,JavaScript,GitHub,FOSS,3D,STEM' >
```

This is somewhat standard HTML behavior and it could be codified and expanded.

In other words, there are a number of ways we could think about cataloging our exhibits. Once we have a good number of exhibits, of course.
